plugin/v2/plugin.go

```diff
@@ -149,13 +149,59 @@ next:
 				} else if !ok {
 					return fmt.Errorf("%q is not settable", s.prettyName())
 				}
-				// is it, so lets update the config in memory
+				// it is, so lets update the config in memory
 				updateConfigEnv(&p.PluginObj.Config.Env, &s)
 				continue next
 			}
 		}
 
-		//TODO: check devices, mount and args
+		// range over all the mounts in the manifest
+		for _, mount := range p.PluginObj.Manifest.Mounts {
```

Comment:
I'm not fond of the code duplication between Env, Mounts and Devices. I'm okay with a follow up PR to fix this. Until then, please add a `TODO` item here.